packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -88,14 +104,19 @@ export const ConnectedRichTextEditor = (props: ConnectedProps) => {
   );
 };
 
-type Props = ConnectedProps & { isInitiallyDisabled: boolean };
-
-const RichTextEditor = (props: Props) => {
-  const { sdk, isInitiallyDisabled, onAction, restrictedMarks, ...otherProps } = props;
+const RichTextEditor = (props: RichTextProps) => {
+  const { sdk, isInitiallyDisabled, onAction, restrictedMarks, onChange, ...otherProps } = props;
   const isEmptyValue = React.useCallback(
     (value) => !value || deepEquals(value, Contentful.EMPTY_DOCUMENT),
     []
   );
+  // TODO: Deprecated use of onChange, should be removed in next major version
```

Comment:
Users won't see this. You should move the notice next to the prop declaration and make it a JS doc e.g.

```typescript
/** 
* @deprecated use `sdk.field.onValueChanged` instead.
*/
onChange?: ...
```